docs/user-documentation/file_viewers.md

```diff
@@ -0,0 +1,84 @@
+# File Viewers
+
+_last updated on 03-12-2020_
+
+
+## What are viewers?
+
+[Viewers](../glossary#viewer) allow site builders to display files in interactive javascript-based widgets, that provide functionality like zooming in/out, turning pages, playing/pausing, viewing in full screen, etc. 
+
+In Drupal, a common way to implement a viewer is through a [module](./glossary/#module) that provides the javascript library, and a field formatter that uses that library. Usually the field formatter can be used on File fields.  
+
+Viewers that are known to work with Islandora 8 include:
+
+* [OpenSeadragon](https://openseadragon.github.io/), via the Drupal module [OpenSeadragon](https://github.com/Islandora/openseadragon) (maintained by the Islandora Foundation)
+* [pdf.js](https://github.com/mozilla/pdf.js), via the Drupal contrib module [PDF](https://www.drupal.org/project/pdf)
+
+Both are included with Islandora Defaults.
+
+## How are viewers configured?
+
+In the generic Islandora 8 content modelling scenario, we often want the viewer to appear on a _"resource node"_ when the file itself is in a field on a Media that references that node. These layers of indirection make displaying viewers on nodes rather complex.
+
+## Example - Open Seadragon (in Islandora Defaults)
+
+Here is how viewers are implemented in Islandora Defaults. This example uses OpenSeadragon, but pdf.js is configured with the same method. This method was chosen because it uses pure configuration and no extra code. It relies on users tagging nodes with appropriate "display hint" terms to trigger Contexts to switch the node's view mode to one that includes an EVA field, which renders the appropriate Media's file in the viewer.
+
+### Expected behaviour for this scenario:
+
+Create a Repository Item, and tag it with "Open Seadragon" (under Display Hints). Add a Media of type "Image" to that node, and tag it with "Original File" (under Media Use). Upload an image file that can be viewed in OpenSeadragon, for example, a jpeg-2000 file, and save (publish) the Media. Go to the Repository Item's page, and you should see your file displayed in the viewer.
+
+
+### Components of this scenario: 
+
+1. A viewer (e.g. the module OpenSeadragon)
```

Comment:
or, more technically correct, a Drupal Field Formatter that supports the file type.

Reply:
Oh, I see that you catch that point below...